mapie/regression.py

```diff
@@ -121,7 +124,10 @@ class MapieRegressor(BaseEstimator, RegressorMixin):  # type: ignore
         Residuals between ``y_train`` and ``y_pred``.
 
     k_: np.ndarray of shape(n_samples_train,)
-        Id of the fold containing each trainig sample.
```

Comment:
Pour améliorer la compréhension, je peux propose un petit réagencement:
```
k_ : np.ndarray 
   - Id of the fold containing each training sample, if cv is not JacknnifeAB. Of shape(n_samples_train,).
   - Dummy array of folds containing each training sample, otherwise. Of shape (n_samples_train, n_resamplings).

Reply:
Fait